webpack.config.js

```diff
@@ -22,6 +22,8 @@ module.exports = {
     "tco-calculator": "./static/js/src/tco-calculator.js",
     "sticky-nav": "./static/js/src/sticky-nav.js",
     imageBuilder: "./static/js/src/imageBuilder.js",
+    lightbox: "./static/js/src/lightbox.js",
```

Comment:
Why do you need to build a standalone version of `lightbox` when its included in applance.js?